api/src/test/java/org/openmrs/validator/ConceptSourceValidatorTest.java

```diff
@@ -9,10 +9,10 @@
  */
 package org.openmrs.validator;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Why? Isn't lang3 the future? ;) I mean "back to the future" was probably better than the sequel but